Makefile.dryice.js

```diff
@@ -654,28 +654,34 @@ function extractCss(callback) {
 }
 
 function extractNls() {
-    var allMessages = {};
+    var defaultData = require(__dirname + "/src/lib/default_english_messages").defaultEnglishMessages;
+
     searchFiles(__dirname + "/src", function(path) {
         if (/_test/.test(path)) return;
         var text = fs.readFileSync(path, "utf8");
-        var matches = text.match(/nls\s*\(\s*("([^"\\]|\\.)+"|'([^'\\]|\\.)+')/g);
+        var matches = text.match(/nls\s*\(\s*("([^"\\]|\\.)+"|'([^'\\]|\\.)+'),\s*("([^"\\]|\\.)+"|'([^'\\]|\\.)+')/g);
         matches && matches.forEach(function(m) {
-            var eng = m.replace(/^nls\s*\(\s*["']|["']$/g, "");
-            allMessages[eng] = "";
+            var match = m.match(/("([^"\\]|\\.)+"|'([^'\\]|\\.)+)/g);      
+            var key = match[0].replace(/["']|["']$/g, "");
+            var defaultString = match[1].replace(/["']|["']$/g, "");
+
+            // If the key not yet in the default file, add it:
```

Comment:
do we have a mechanism to keep "default_english_messages" up-to-date with the used keys?

Reply:
oh I see following line with `fs.writeFileSync`

Reply:
if you run `node makefile.dryice.js nls` the file will get updated with new keys, maybe we should include that in our build process but not sure where 🤔